patches/glog-tsan-annotations.patch

```diff
@@ -1,5 +1,7 @@
---- src/glog/vlog_is_on.h.in 2015-03-11 07:02:27.000000000 +0300
```

Comment:
Do the timestamps here serve a purpose/for tracking?

Reply:
I dont think so. All other patch files do not use this pattern which is why i switched to the new one.
We should just be using git history for tracking anyway.